eslint.config.js

```diff
@@ -30,15 +27,7 @@ module.exports = [
             }
         },
         rules: {
-            "n/no-unpublished-require": "off",
-            "n/no-extraneous-require": [
-                "error",
-                {
-                    allowModules: [
-                        "@octokit/rest"
-                    ]
-                }
-            ]
```

Comment:
We're not using `@octokit/rest` in tests anymore, so there's no need to override the config for this rule.